domains/af.json

```
@@ -0,0 +1,25 @@
+{
+  "owner": {
+    "username": "afahmygit",
+    "email": "git@ahmedfahmy.com"
+  },
+  "records": {
+    "CNAME": ["cv.ahmedfahmy.com"],
+    "MX": [
+      {
+        "priority": 10,
+        "target": "mailing.ahmedfahmy.com"
+      }
+    ],
```

default priority is 10
```suggestion
    "MX": [ "mailing.ahmedfahmy.com" ],
```